livekit_analytics.proto

```diff
@@ -152,3 +153,27 @@ message AnalyticsEvent {
 message AnalyticsEvents {
   repeated AnalyticsEvent events = 1;
 }
+
+message AnalyticsRoomParticipant {
+  string id = 1;
+  string identity = 2;
+  string name = 3;
+  ParticipantInfo.State state = 4;
+  ParticipantInfo.Kind kind = 5;
```

Comment:
nbd, but do we need name and kind? is the idea to reconstruct things from here?

Reply:
thats right, if there are missed join/active event and participant was still active, we could reconstruct participant with the data